### PR TITLE
Added-create data model

### PR DESCRIPTION
**Pull request by moise10r**
### In this project, I did the following thing
- I referred to the ERD diagram from the sneak peek if you have any questions about table or column names or data relationships.

   - Build your project schema.
   - Create and run the necessary migration files.
    - Table and column names should match the ERD diagram.
    - note: photo for users table should be a link to an image
   - Foreign keys should be included.
   - All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: db/migrate/20211030114756_add_user_to_posts.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUserToPosts < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```
Comment (sinansevgi) [inline db/migrate/20211030114756_add_user_to_posts.rb:3]: - [x] According to the ERD scheme, you should refer to the **users** _as_ **authors** for **posts**, **likes**, and **comments**. Please check the ERD schema and update your models accordingly. You can take my change suggestion as an example.  :arrow_down: 

```suggestion
add_reference :posts, :author, index: true
add_foreign_key :posts, :users, column: :author_id
```

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
------

Hi @moise10r , 

Good job so far!
Most of the relations configured correctly :+1: 
There are some issues that you still need to work on to go further but you are almost there!


 ## Required Changes ♻️
_Please check the inline comments for my suggestions_ :arrow_down:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

------

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by moise10r** [comment]
Just the changes are reflected in schema.rb in the dB folder so for that I generated new migration files for changing user_id into author_id

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @moise10r, 

Great Work :+1: 

The author column has the correct name now :clap:
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by moise10r** [comment]
Thank you for taking the time to review this PR.

**Pull request merged**